updater/scripts/git-download.sh

```diff
@@ -19,7 +19,7 @@ if ghe_greater_equal "2.11.0" ; then
     # upcoming version. In the meantime, we grep for all log entries in the two
     # most recent log files (because the information from yesterday may or not
     # be rotated already).
-    CAT_LOG_FILE="zcat -f /var/log/github-audit.log{,.1*} | grep -F '$(date --date='yesterday' +'%b %d')'"
+    CAT_LOG_FILE="zcat -f /var/log/github-audit.log{,.1*} | grep -F '$(date --date='yesterday' +'%b %_d')'"
```

Comment:
Your change looks correct, and I tested that it works successfully 😃.

However, looking at the rest of this line now, I find the nested `'` syntactically tricky. For instance, attempting to run the `CAT_LOG_FILE` commands on the shell results in an error for me:

```
grep: %d): No such file or directory
```

While `CAT_LOG_FILE` works well with `eval`, we might want to make the syntax more robust should we, for instance, draw the command out of the `CAT_LOG_FILE` at some point.

Reply:
Agreed. Let's handle that in a subsequent PR!

Reply:
Okay, then let’s merge this one first 😃.